srcpkgs/autogen/template

```diff
@@ -1,31 +1,35 @@
 # Template file for 'autogen'
 pkgname=autogen
-version=5.18.4
-revision=7
+version=5.18.16
+revision=1
 build_style=gnu-configure
-configure_args="--disable-static"
-hostmakedepends="pkg-config perl"
-makedepends="guile guile-devel libxml2-devel gc-devel"
+configure_args="--disable-static --disable-dependency-tracking --without-libregex"
+hostmakedepends="guile pkg-config perl"
+makedepends="guile guile-devel libatomic_ops-devel libxml2-devel gc-devel"
 depends="guile perl"
-short_desc="The Automated Program Generator"
-maintainer="Orphaned <orphan@voidlinux.org>"
-license="GPL-3"
+short_desc="Automated Program Generator"
+maintainer="Juan RP <xtraeme@voidlinux.org>"
```

Comment:
I'm pretty sure @xtraeme don't like this change

Reply:
woops, I didnt add that part, thanks, will fix